src/styles.css

```diff
@@ -12,3 +12,10 @@ html {
 body {
     background-color: #F8F8F8;
 }
+
+.showNotification {
+    max-width: 400px;
+    position: static;
```

Comment:
I'm sorry. This had to be `fixed`. I told you the wrong one to use. I'll fix it in a commit to this PR.